grammars/quotation/grammar.js

```diff
@@ -49,6 +55,9 @@ module.exports = grammar(base, {
 
     quotation_type_expander: ($) => seq("$", "ty", ":", $.quotation_variable),
 
+    quotation_top_expander: ($) =>
+      seq("$", "top"),
+
```

Comment:
Is there any reason to disable capturing a toplevel definition into a quotation_variable here?